### ISSUE TITLE
IndexOutOfRangeException: Array index is out of range.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [coolboy1/dotween#32]
```
Hello! I have this exception 
in 1.0.030 DOTween 
in Unity 5 Version 5.0.0f4 (5b98b70ebeb9)
This happens in my method MyMethod in this order 

MyMethod () // workds
// one gazillion other tweens
// some time
DOTween.Clear(true)
MyMethod () // crashes

IndexOutOfRangeException: Array index is out of range.
(wrapper stelemref) object:stelemref (object,intptr,object)
DG.Tweening.Core.TweenManager.AddActiveTween (DG.Tweening.Tween t) (at 
D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween.Assembly/DOTween/Core/Tween
Manager.cs:661)
DG.Tweening.Core.TweenManager.GetTweener[Single,Single,FloatOptions] () (at 
D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween.Assembly/DOTween/Core/Tween
Manager.cs:95)
DG.Tweening.DOTween.ApplyTo[Single,Single,FloatOptions] 
(DG.Tweening.Core.DOGetter`1 getter, DG.Tweening.Core.DOSetter`1 setter, Single 
endValue, Single duration, DG.Tweening.Plugins.Core.ABSTweenPlugin`3 plugin) 
(at 
D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween.Assembly/DOTween/DOTween.cs
:785)
DG.Tweening.DOTween.To (DG.Tweening.Core.DOGetter`1 getter, 
DG.Tweening.Core.DOSetter`1 setter, Single endValue, Single duration) (at 
D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween.Assembly/DOTween/DOTween.cs
:228)
GameInsight.uGui.PreloaderLogo..ctor (UnityEngine.UI.Image image) (at 
Assets/_Develop/code/GameInsight/uGui/PreloaderLogo.cs:17)
```

Original issue reported on code.google.com by `pavel.ol...@gmail.com` on 13 Mar 2015 at 12:43

**Comment by GoogleCodeExporter** [comment]
```
Hi Pavel,

this is a very weird issue that was reported a couple times, but then 
disappeared and I was never able to reproduce it. Now that it resurfaced for 
you, I'd like to ask you a couple things because I really want to fix it once 
and for all:

1) Can you change DOTween.Clear(true) to DOTween.Clear(false) and let me know 
if this happens again? If it doesn't, it will help me a lot to pinpoint the 
issue.

2) If it still happens, can you upgrade to the version I've attached here, so 
we have the same lines in case of error logs, and write me again the log result?

Thanks & sorry for this problem.
```

Original comment by `daniele....@gmail.com` on 13 Mar 2015 at 6:10
- Changed state: **DataNeeded**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

Attachments:
- [DOTween_1_0_305.zip](https://storage.googleapis.com/google-code-attachments/dotween/issue-32/comment-1/DOTween_1_0_305.zip)

**Comment by GoogleCodeExporter** [comment]
```
Hi Pavel,

should you manage to get more info on this matter, I moved DOTween's Issues to 
GitHub (since Google Code is shutting down).

https://github.com/Demigiant/dotween/issues/2
```

Original comment by `daniele....@gmail.com` on 18 Mar 2015 at 6:50

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.0.336

https://github.com/Demigiant/dotween/issues/2
```

Original comment by `daniele....@gmail.com` on 23 Mar 2015 at 8:32
- Changed state: **Fixed**
